.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -5,9 +5,9 @@ contact_links:
   about: If you have questions, please post them in our GitHub Discussions forum.
 - name: 📝 Contributing Guidelines
   url: https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md 
-  about: Please review our CONTRIBUTING.md file for guidelines. 
-- name: ✉️ Contact Us
-  url: mailto:quire@getty.edu
-  about: Need assistance? Send us an email. 
+  about: Please take time to review our CONTRIBUTING.md file. 
+- name: 🖥️ Website 
+  url: https://quire.getty.edu
+  about: Visit our website to learn more about Quire and find contact information.
```

Comment:
I don't like "and find contact information."

> Visit our website to learn more about Quire, or email us at quire@getty.edu.

?

Or just

> Visit our website to learn more about Quire.

Reply:
You're right. Ok, I'm going to forgo email. I think it's a bit tricky to give them that direct lifeline anyways.

Reply:
ah shoot, once again it disregarded your approval... can you approve again? I swear I'm not intentionally dismissing you.